wg-infra/README.md

```diff
@@ -0,0 +1,43 @@
+# Infrastructure WG
+
+Oversees all infrastructure, bots, automation, services and systems that support the Electron maintainers group.
+
+## Membership
+
+| Avatar | Name | Role | Time Zone |
+| -------------------------------------------|----------------------|----------------------------| -------- |
+| <img src="https://github.com/marshallofsound.png" width=100 alt="@marshallofsound">  | Samuel Attard [@MarshallOfSound](https://github.com/marshallofsound) | Member | PT (Vancouver) |
+| <img src="https://github.com/codebytere.png" width=100 alt="@codebytere">  | Shelley Vohr [@codebytere](https://github.com/codebytere) | Member | CET (Berlin) |
+| <img src="https://github.com/jkleinsc.png" width=100 alt="@jkleinsc">  | John Kleinschmidt [@jkleinsc](https://github.com/jkleinsc) | Member | ET (Harrisburg) |
+
+## Areas of Responsibility
+
+* Cloud Services
+  * Heroku
+  * AWS
+  * Azure
+  * GitHub
+  * Grafana
+  * Goma
+  * CircleCI
+  * Appveyor
+  * Sentry
+  * Slack
+  * 1Password
+  * NPM
+* Electron Bots
```

Comment:
We should be more specific about which bots

Reply:
All the bots? Should I list them all 😅 Or maybe it's safer to say we own the infrastructure the bots run on, and deployments, but not the code itself?

Reply:
That seems slightly better - let's do that